### ISSUE TITLE
Support for task priority or weight?

### ISSUE THREAD

**Issue by charleswhchan** [closeio/tasktiger#96]
Curious if there is any plan in the roadmap for TaskTiger to support tasks with different priorities or weights? 

Thank you 😃

**Comment by thomasst** [comment]
Do you have a specific use case? How exactly should it work?

**Comment by slidenerd** [comment]
Yes a cron job that fetches 1000 urls every 15 mins should always work even when we have 1000 emails to send to the user, as the email one receives lower priority

**Comment by alanhamlett** [comment]
I think sub-queues would be a good solution for your use case.
